FusionIIIT/templates/dashboard/navbar.html

```diff
@@ -1,138 +1,106 @@
 {% load static %}
-
 <!-- navbar -->
-
 {% block navBar %}
-    {% include 'dashboard/sidenavbar.html' %}
-
-    <div>
-        <div class="ui stackable massive inverted top attached menu" id="navbar">
-            <a class="item" id="sidebartrigger">
-                <i class="sidebar icon"></i>
+{% include 'dashboard/sidenavbar.html' %}
```

Comment:
Why is this file included in the PR? Please remove irrelevant changes.

Reply:
Changed Massive to Large menu.